packages/components/tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
     "extends": "../../tsconfig.json",
     "compilerOptions": {
+        "paths": {
+            "@trezor/theme": ["./packages/theme/src/*"]
+        },
```

Comment:
What does this do?

Reply:
auto import was always importing `packages/theme/src` for me, which was causing build errors. This is just an alias to tell TS it should not do it. But it's not the case with the other packages and I didn't send time to figure out why. Could you please check on some other branch if auto imports from `theme` to `components` work fine for you?

Reply:
Yeah it happens to me as well. 😕